twilight-model/src/application/monetization/sku.rs

```diff
@@ -0,0 +1,75 @@
+use serde::{Deserialize, Serialize};
+
+use crate::id::{
+    marker::{ApplicationMarker, SKUMarker},
+    Id,
+};
+
+use super::{SKUFlags, SKUType};
+
+/// SKUs (stock-keeping units) in Discord represent premium offerings that can be made available to your application's users or guilds.
+#[derive(Clone, Debug, Eq, Hash, PartialEq, Serialize, Deserialize)]
+pub struct SKU {
```

Comment:
this should be named `Sku`, following rust's naming conventions for abbreviations

Reply:
Probably should also do the same with SKUFlags, SKUMarker, & SKUType